src/Ledger/PDF.lagda

```diff
@@ -68,4 +70,11 @@ open import Ledger.Chain
 \section{Properties}
 \input{Ledger/Utxo/Properties}
 
+\clearpage
+
+\appendix
+
+\section{Appendix: Agda essentials}
```

Comment:
I think this should go into the `EssentialAgda` file (I know that `Properties` above has the same issue, and we should fix that at some point as well).

Reply:
I'm assuming you meant that, of the four lines 74--77, only line 77 should be moved to `EssentialAgda.lagda`.  That's what will take effect in my next commit to this PR.